sessionctx/variable/session.go

```diff
@@ -426,6 +429,7 @@ func NewSessionVars() *SessionVars {
 		CommandValue:                uint32(mysql.ComSleep),
 		TiDBOptJoinReorderThreshold: DefTiDBOptJoinReorderThreshold,
 		SlowQueryFile:               config.GetGlobalConfig().Log.SlowQueryFile,
+		LowResolutionTSO:            false,
```

Comment:
no need to set it to false explicitly, golang init it with false automatically.

Reply:
Fixed